Signal/src/ViewControllers/CallViewController.swift

```diff
@@ -1125,4 +1137,13 @@ class CallViewController: OWSViewController, CallObserver, CallServiceObserver,
         updateLocalVideoTrack(localVideoTrack: localVideoTrack)
         updateRemoteVideoTrack(remoteVideoTrack: remoteVideoTrack)
     }
+
+    // MARK: - Event Handlers
+
+    func leaveCallViewTapped(sender: UIGestureRecognizer) {
```

Comment:
This event handler isn't wired up in this PR, but this is how we'll eventually have "text during call".

Reply:
Done.